plivo/rest/client.py

```diff
@@ -16,14 +16,18 @@
                              Numbers, Pricings, Recordings, Subaccounts, CallFeedback)
 from plivo.resources.live_calls import LiveCalls
 from plivo.resources.queued_calls import QueuedCalls
+from plivo.resources.regulatory_compliance import EndUsers, ComplianceDocumentTypes, ComplianceDocuments, \
+    ComplianceRequirements, ComplianceApplications
 from plivo.utils import is_valid_mainaccount, is_valid_subaccount
 from plivo.version import __version__
 from requests import Request, Session
 
 AuthenticationCredentials = namedtuple('AuthenticationCredentials',
                                        'auth_id auth_token')
 
-PLIVO_API = 'https://api.plivo.com'
+# PLIVO_API = 'http://localhost:5000'
+# PLIVO_API = 'https://api.plivo.com'
```

Comment:
Hope you have already noted but adding comment so you can do cleanUp before the final merge.